### PR TITLE
Add assert API

### PR DESCRIPTION
**Pull request by Zhigalov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
 npm install http-assert-value --save
 ```
 
-### Example
+## Usage
 ```js
 const assert = require('http-assert-value');
 
+// Common assert interface
+assert(+0 === -0, 'Zeros is not equal', 500, 'ZNE', {
```
Comment (YuuKohaku) [inline README.md]: is -> are

File: README.md [hunk 2]
```diff
@@ -29,3 +38,76 @@ try {
     error.options;    // { value: 'DROP DATABASE BOOKS;', internalCode: '400_TVI' }
 }
 ```
+
+## API
+
+### assert(value, message, code, shortMessage, options={})
+Assert value with http-specific error data
+
+* value `...` - asserting value
```
Comment (YuuKohaku) [inline README.md]: Any?

File: README.md [hunk 3]
```diff
@@ -29,3 +38,76 @@ try {
...
+* message `String` - error text
+* code `Number` - error http-code
+* shortMessage `String` - error three-letters code
```
Comment (YuuKohaku) [inline README.md]: Скорее, three-letter error code

File: README.md [hunk 4]
```diff
@@ -29,3 +38,76 @@ try {
...
+### assert.tryText(value, field = 'Text')
+Assert text, like search request, when value is defined
+
+* value `String|Array<String>` - asserting value
+* \[field\] `String` - name of the parameter containing value
+
+### assert.tryBySchema(value, schema, options = {})
+Assert object by schema, when value is defined
+
+* value `...|Array<...>` - asserting value
```
Comment (YuuKohaku) [inline README.md]: Any?

**Review by YuuKohaku — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Zhigalov) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (Zhigalov) [reply README.md]: Rename `...` -> `Any`

File: README.md (hunk 3 above)
Comment (Zhigalov) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (Zhigalov) [reply README.md]: Rename `...` -> `Any`

**Pull request merged**